data-science-onramp/feature-engineering/citibike.ipynb

```diff
@@ -0,0 +1,385 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feature Engineering with Pandas and scikit-learn\n",
+    "\n",
+    "This notebook demonstrates how to use AI Platform notebooks to perform feature engineering on a dataset using `pandas`.\n",
+    "\n",
+    "You will load the data into a `Pandas DataFrame`, clean up the columns into a useable format, and then restructure the data into feature and target data columns. \n",
+    "\n",
+    "Before you jump in, let's cover some of the different tools you'll be using:\n",
+    "\n",
+    "+ [AI Platform](https://cloud.google.com/ai-platform) consists of tools that allow machine learning devlopers and data scientists to run their ML projects quickly and cost-effectively.\n",
+    "\n",
+    "+ [Cloud Storage](https://cloud.google.com/storage/) is a unified object storage for developers and enterprises, from live data serving to data analytics/ML to data archiving.\n",
+    "\n",
+    "+ [Cloud SDK](https://cloud.google.com/sdk/) is a command line tool which allows you to interact with Google Cloud products. This notebook introduces several `gcloud` and `gsutil` commands, which are part of the Cloud SDK. Note that shell commands in a notebook must be prepended with a `!`.\n",
+    "\n",
+    "+ [Pandas](https://pandas.pydata.org/) is a data analysis and manipulation tool built on top of the Python programming language."
```

Comment:
should scikit-learn also be mentioned here given that it's in the title?

Reply:
Good point. I actually have not used scikit-learn, so I am removing it from the title. If I use it for the other datasets, I will add it in the title and in the list of tools.